src/components/Footer/Footer.js

```diff
@@ -42,62 +46,95 @@ const Footer = ({
   t,
   dispatchSetLanguage,
   dispatchSetProjection,
-}) => (
-  <RSFooter className="wkp-footer">
-    <div className="wkp-footer-left">
-      <Copyright
-        layerService={layerService}
-        format={f => `${t('Geodaten')} ${f}`}
-      />
-    </div>
+}) => {
+  const [isOpen, setIsOpen] = useState(null);
```

Comment:
Usually the setter of a isXXX  variable is called `setOpen`  .

Reply:
Usally a isXXXX variable means it's a boolean , which is not the case here, call it just `open` or  `dialogOpen` whatever